contracts/MasterChefV2.sol

```diff
@@ -42,11 +43,11 @@ contract MasterChefV2 {
     UbiquityAlgorithmicDollarManager public manager;
 
     // uGOV tokens created per block.
-    uint256 public uGOVPerBlock = 1e18;
+    uint256 public uGOVPerBlock;
```

Comment:
if it is not set there it could be setup in the constructor

Reply:
we will deploy with `uGOVPerBlock = 0`